internal/fuzzbinder/fuzzscraper.go

```diff
@@ -263,12 +263,13 @@ func getFuzzEffortFromFile(reader io.Reader, revisionDigest intoto.DigestSet) (*
 		if err := lineScanner.Err(); err != nil {
 			return nil, err
 		}
-		fileFuzzEffort, err = getFuzzStatsFromFile(lineScanner)
+		gotFuzzEffort, err := getFuzzStatsFromFile(lineScanner)
```

Comment:
Here you could just return 

```
return getFuzzStatsFromFile(lineScanner)
```

The function will be a lot simpler, and no separate variable declaration will be needed.

Reply:
I agree but two returns are needed. One if `isGoodHash` and another in case it is not a good hash and we want zeros. I updated it to remove the variable declaration.